DependencyInjection/CmfSeoExtension.php

```diff
@@ -66,6 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadPhpcr($config['persistence']['phpcr'], $loader, $container);
 
             $loader->load('matcher_phpcr.xml');
+            $loader->load('phpcr-sitemap.xml');
```

Comment:
but maybe both this line and the line above could go to into $this->loadPhpcr to keep load() shorter